src/components/Frame/components/Toast/Toast.scss

```diff
@@ -4,7 +4,7 @@ $Backdrop-opacity: 0.88;
 .Toast {
   @include text-style-display-small;
   display: inline-flex;
-  max-width: rem(400px);
+  max-width: rem(500px);
```

Comment:
This was changed to 720px, but I'm not sure the width needs to be increased that much. If the content guidelines suggest a message no longer than 3 words, does it really need to exceed 500px?

![screen shot 2019-02-12 at 5 32 21 pm](https://user-images.githubusercontent.com/6239480/52680117-35b93300-2eec-11e9-8762-004eb70d5034.jpg)

Reply:
This seems fine to me. `720px` seems pretty wide and probably unnecessary for most cases.

Reply:
What happens on small screens, would this be more useful at ‘100vw’ ?

Reply:
I recently learned that vw and vh don't work well in Safari. On small screens, a long toast message wraps.